# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavcodec/moz.build
# Commit: 74fa25299c53
# Full Hash: 74fa25299c53a238c11fa559f0efc7038518b5c2
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-28 16:14:38
# Regressor Bug: 1879873
# File Overlap Count: 1
# Description:
#   Bug 1879873 - Remove MOZ_LIBAV_FFT. r=karlt
#   
#   We have some time to use av_fft.h, after which we'll have to use tx.h directly.
#   It's not hard to update.
#   
# ==============================================================================

diff -r cff6a02ad4ee -r 74fa25299c53 media/ffvpx/libavcodec/moz.build
--- a/media/ffvpx/libavcodec/moz.build	Wed Feb 28 12:50:26 2024 +0000
+++ b/media/ffvpx/libavcodec/moz.build	Wed Feb 28 12:50:26 2024 +0000
@@ -22,6 +22,7 @@
     'allcodecs.c',
     'avcodec.c',
     'avdct.c',
+    'avfft.c',
     'avpacket.c',
     'bitstream.c',
     'bitstream_filters.c',
@@ -145,11 +146,6 @@
   CXXFLAGS += CONFIG["MOZ_LIBVPX_CFLAGS"]
   OS_LIBS += CONFIG["MOZ_LIBVPX_LIBS"]
 
-if CONFIG['MOZ_LIBAV_FFT']:
-    SOURCES += [
-        'avfft.c',
-    ]
-
 SYMBOLS_FILE = 'avcodec.symbols'
 NoVisibilityFlags()
 